### DISCUSSION TITLE
请问jsDelivr版是离线版吗？

### DISCUSSION THREAD

**Discussion by f59375543** [the1812/Bilibili-Evolved#3892] [提问 / Q&A]
我发现从上个月开始，我使用jsDelivr版，打开b站动态页，都会卡顿很久，甚至状态框都刷不出来，不得不刷新页面好几次才能完整加载出来

以前用离线版没有出现过这种情况

**Comment by the1812** [comment]
不是, 只是一个国内能直连的 CDN, 但是速度算不上快
